### ISSUE TITLE
Stage: Visual Focus order is missing in Sign in page

### ISSUE THREAD

**Issue by gopal2020** [NYCPlanning/labs-zap-search#1031]
Staging Environment:

Actual Result: Visual Focus order is missing in Sign in page

Expected Result: The focus is visible while navigating from top to bottom in the page using the tab key.
**Note:  WCAG 2.0 Guideline, Success Criteria 2.1.1- Visual Focus**

![image](https://user-images.githubusercontent.com/57265341/69980968-5431fe80-14ff-11ea-972a-0dd154424c6e.png)

**Comment by allthesignals** [comment]
We don't control this page. You'd have to take it up with DoiTT.

**Comment by hannahkates** [comment]
Will report to NYC.ID
